metropolis-font/SIL Open Font License.txt

```diff
@@ -0,0 +1,43 @@
+Copyright (c) 2015, Chris Simpson, with Reserved Font Name: "Metropolis".
```

Comment:
Good one, since you added the license file. Which make sure your are following all complainces

Reply:
I think we are only going to use Metropoli-Regular font, do I remove all the other files?